src/engines/engine262.js

```diff
@@ -34,12 +34,12 @@ class Engine262Installer extends Installer {
   }
 
   async install() {
-    const bin = path.join(this.installPath, 'package', 'bin', 'engine262.js');
+    const bin = path.join(this.installPath, 'package', 'lib', 'engine262.js');
     this.binPath = await this.registerBinarySymlink(bin, 'engine262');
   }
 
   async test() {
-    const program = 'print("42");';
```

Comment:
all these engines should have `print` in the cli, it's used in test262 and people expect to use it with esvu as well.